mono/mini/mini-wasm-debugger.c

```diff
@@ -620,11 +620,12 @@ static gboolean describe_value(MonoType * type, gpointer addr)
 			MonoString *str_obj = *(MonoString **)addr;
 			if (!str_obj)
 				mono_wasm_add_string_var (NULL);
```

Comment:
Both sides of the if-then should have a {} if one of them has it.

Reply:
Done! :)